src/main/java/org/opentripplanner/middleware/tripmonitor/jobs/TripAnalyzer.java

```diff
@@ -60,6 +62,18 @@ public void run() {
                     continue;
                 }
 
+                // Refetch the trip from the database. This to ensure the trip has any updates made to the trip between
```

Comment:
This -> "This is"

Reply:
Fixed in 6948fab3bf5ead8b87a4b47676a38fe9a6e1691f